### PR TITLE
Add discussion of Cluster Network Operator (CNO)

### PR DESCRIPTION
**Pull request by jboxman**
Based on DNS Operator and ingress controller content.

### CODE REVIEW

File: modules/nw-cno-view.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  Self Link:           /apis/operator.openshift.io/v1/networks/cluster
+Spec:
+  Cluster Network:
+    Cidr:         10.128.0.0/14
+    Host Prefix:  23
+  Default Network:
+    Type:  OpenShiftSDN
+  Service Network:
+    172.30.0.0/16
+Status:
```
Comment (squeed) [inline modules/nw-cno-view.adoc]: It would be good to discuss the status field as well.

Comment (jboxman) [reply modules/nw-cno-view.adoc]: @squeed @danwinship, what can I say about the `Status` field?

I'd like to have this ready by EOB on Friday for the GA next week. Any information you can provide is appreciated. Thanks!

**Comment by squeed** [comment]
The Status field is the network operator's understanding of the current deployed network configuration. It includes all active IP address ranges, along with the observed pod network MTU.

It is possible for the Status and Spec to be different. If the operator is rolling out a change, it will update Status when that rollout has happened. Developers and administrators writing automated tooling that needs to consume IP address blocks should almost always get that from Status.

**Comment by jboxman** [comment]
@squeed, would I want to also include `oc describe network.config/cluster` or use that instead of `network.operator/cluster`? The former includes a populated `Status`, but the latter does not.

**Comment by jboxman** [comment]
Hi @bmeng @zhaozhanqi

Can you PTAL at this? I'd like to get it merged for 4.1 GA.

Thank you!

**Comment by bmeng** [comment]
LGTM
Thanks.

**Comment by jboxman** [comment]
@openshift/team-documentation, PTAL? Thanks!

File: modules/nw-cluster-network-operator.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+. Run the following command to view the state of the Cluster Network Operator:
++
+----
+$ oc get clusteroperator/network
+
+NAME      VERSION      AVAILABLE   PROGRESSING   DEGRADED   SINCE
+network   4.1.0-rc.3   True        False         False      50m
+----
+The following fields provide information about the status of the operator:
```
Comment (ahardin-rh) [inline modules/nw-cluster-network-operator.adoc]: s/operator/Operator
PROGRESSING`, and `DEGRADED` (Oxford comma)

File: modules/nw-cno-logs.adoc [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+// * networking/cluster-network-operator.adoc
+
+[id="nw-cno-logs_{context}"]
+= Viewing Cluster Network Operator logs
+
+You can view Cluster Network Operator logs by using the `oc logs` command.
+
+.Procedure
+
+* Run the following command to view the logs of the Cluster Network Operator:
```
Comment (ahardin-rh) [inline modules/nw-cno-logs.adoc]: Even if it's just one step, it should be a numbered step

. Run

File: modules/nw-cno-logs.adoc [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+----
```
Comment (ahardin-rh) [inline modules/nw-cno-logs.adoc]: Missing the + on the line before the command

File: modules/nw-operator-cr.adoc [hunk 5]
```diff
@@ -85,49 +78,58 @@ then you must set this explicitly to 50 less than the smallest node MTU value.
...
-<5> {kube-proxy}
-<6> {kube-proxy-ip}
-<7> {kube-proxy-refresh}
-<8> {iptables-min-sync-period}
+<5> The isolation mode for the OpenShift SDN CNI plug-in.
+
+<6> MTU for the VXLAN overlay network. This value is normally configured
+automatically.
+
+<7> The port to use for all VXLAN packets. The default value is 4789.
```
Comment (ahardin-rh) [inline modules/nw-operator-cr.adoc]: `4789`

**Comment by ahardin-rh** [comment]
@jboxman Just a few minor comments from me. Otherwise, looks good! :+1:

File: modules/nw-cno-logs.adoc (hunk 3 above)
Comment (jboxman) [reply modules/nw-cno-logs.adoc]: @ahardin-rh, I think I've seen conflicting information on whether it's numbered or not.

Comment (jboxman) [reply modules/nw-cno-logs.adoc]: For mod-docs, we're using bullets.

Comment (ahardin-rh) [reply modules/nw-cno-logs.adoc]: @jboxman Yeah, disregard. I missed that. Sorry! :woman_facepalming:

**Comment by jboxman** [comment]
I'm going to squash and then merge this. A different PR will fix-up the TOC as this supersedes _Configuring the network_ topic.

**Pull request merged**